docker-compose.yml

```diff
@@ -64,8 +64,10 @@ services:
         source: .
         target: /app
   applicants:
-    build: ./bops-applicants
-    entrypoint: ./docker-entrypoint.sh
+    build:
+      context: ./bops-applicants/docker/ruby
+      dockerfile: Dockerfile
+    entrypoint: /home/rails/bops-applicants/docker/ruby/docker-entrypoint.sh
```

Comment:
Looking at the bops-applicants docker file https://github.com/unboxed/bops-applicants/blob/main/docker/ruby/Dockerfile#L31 `/bin/docker-entrypoint.sh` might work here too?

Reply:
This makes sense but doesn't work for me hm

Reply:
I don't think it's worth trying to figure out why, in that case; let's just go with this as it is.